sycl/plugins/opencl/pi_opencl.cpp

```diff
@@ -888,6 +923,57 @@ pi_result piContextGetInfo(pi_context context, pi_context_info paramName,
     std::memcpy(paramValue, &result, sizeof(cl_bool));
     return PI_SUCCESS;
   }
+  case PI_CONTEXT_INFO_ATOMIC_MEMORY_ORDER_CAPABILITIES: {
```

Comment:
From SYCL 2020 spec, context info query:

> 	Returns the set of memory orderings supported by atomic operations on all devices in the context, which is guaranteed to include relaxed.

I suggest that we instead implement `context` query at SYCL RT level, because it will be much simpler there to perform device-specific queries and then intersect results.